action.yml

```diff
@@ -58,9 +58,9 @@ runs:
         GO111MODULE: off
```

Comment:
do we need to enable gomod?

Reply:
I think we should enable gomod by default now, as we are no longer using the `go get` functionality. I assume it was kept disabled for getting the dependencies using go get itself

Reply:
> do we need to enable gomod?

Yes, thank you for catching this.  Updated.  You can see a successful run on https://github.com/samuelkarp/go-runc/pull/1.